_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -143,6 +143,22 @@ This link has [visible][] text with mathematical symbols, that does not match th
 <a href="/" aria-label="Proof of two multiplied by two is four">Proof of 2&times;2=4</a>
 ```
 
+#### Failed Example 4
+
+This link has [visible][] text does not match the [accessible name][] because there is a hyphen in the accessible name. 
+
+```html
+<a href="#" aria-label="non-standard">nonstandard</a>
+```
+
+#### Failed Example 5
+
+This link has [visible][] text does not match the [accessible name][] because there are extra spaces in the accessible name. 
+
+```html
+<a aria-label="Call 1 2 3. 4 5 6. 7 8 9 0." href="tel:1234567890">123.456.7890</a>
```

Comment:
Was the inclusion of a full stop at the end of the aria-label attribute value deliberate? If so, do we need to provide clarification regarding its potential impact on the outcome? In the issue description only extra spaces are mentioned.

Reply:
thanks! I edited to the aria-label.